AdyenDropIn/DropInComponent.swift

```diff
@@ -132,11 +132,13 @@ public final class DropInComponent: NSObject, PresentableComponent {
         component.environment = environment
         
         switch component {
+        case let component as PreApplePayComponent:
+            component.presentationDelegate = self
+            navigationController.present(asModal: component)
         case let component as PresentableComponent where component.requiresModalPresentation:
             navigationController.present(asModal: component)
-        case let component as PresentableComponent:
-            let exceptions = (component.viewController is UIAlertController) || (component is ApplePayComponent)
-            navigationController.present(component.viewController, customPresentation: !exceptions)
+        case let component as PresentableComponent where component.viewController is UIAlertController:
+            navigationController.present(component.viewController, customPresentation: true)
```

Comment:
`customPresentation` should be false, right?

Reply:
yes, you are correct 👍